index.html

```diff
@@ -41,6 +41,7 @@ <h1>🌱 Script Seed</h1>
             </div>
             <div class="col-sm text-right">
                 <a id="downloadButton" class="btn btn-primary" role="button" target="_blank"><i class="fas fa-download"></i></a>
+                <a id="copyButton" class="btn btn-primary" role="button" target="_blank"><i class="far fa-copy"></i></a>
```

Comment:
This button does not copy to the clipboard, it's just a copy of the download button.